x/metadata/client/cli/tx.go

```diff
@@ -155,6 +156,58 @@ func RemoveScopeCmd() *cobra.Command {
 	return cmd
 }
 
+func AddRemoveScopeDataAccessCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "scope-data-access [add|remove] [scope-id] [data-access]",
```

Comment:
For the `Use` lines, the documentation says:

> Use is the one-line usage message. Recommended syntax is as follow:
  [ ] identifies an optional argument. Arguments that are not enclosed in brackets are required.
  ... indicates that you can specify multiple values for the previous argument.
  |   indicates mutually exclusive information. You can use the argument to the left of the separator or the
      argument to the right of the separator. You cannot use both arguments in a single use of the command.
  { } delimits a set of mutually exclusive arguments when one of the arguments is required. If the arguments are
      optional, they are enclosed in brackets ([ ]).
Example: add [-F file | -D dir]... [-f format] profile

```suggestion
		Use:   "scope-data-access {add|remove} scope-id data-access",
```

I updated the other `Use` lines in this file in my other PR.

Reply:
Got it, I was following what we had before.  We have this all over the place with the [] on required args.